test/MorseL.Shared.Tests/MorseL.Shared.Tests.csproj

```diff
@@ -10,10 +10,10 @@
     <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel" Version="2.0.0" />
```

Comment:
We ought to introduce a solution level Directory.Build.Props for these versions:
https://github.com/berdon/Orleans.Providers.Redis/blob/master/Directory.Build.props

Reply:
Agreed. I made #51 to do that.